ServerCore/Pages/Index.cshtml

```diff
@@ -6,13 +6,13 @@
 
 <h3>Our next event</h3>
 
-<h4>Microsoft Non-Intern Puzzleday 2021</h4>
+<h4>Microsoft Intern Puzzleday 2021: 2022 Edition</h4>
 <table>
     <tr>
         <td style="width:75%;background-image:url('/images/lightpaperfibers.png');padding:6px;border:solid;border-color:lightgrey;border-width:0.5px">
-            <a href="https://puzzlehunt.azurewebsites.net/nipd2021/play" style="text-decoration:none;">
+            <a href="https://puzzlehunt.azurewebsites.net/pd2021.2203/play" style="text-decoration:none;">
                 <div style="font-family:'Book Antiqua',Garamond,'Times New Roman';font-size:48px;text-transform:uppercase;text-align:center;color:black;border-bottom:solid;border-width:3px;">
-                    Non-Intern Puzzleday 2021<br />
+                    Intern Puzzleday 2021 ver. 2022<br />
```

Comment:
Why 2022 and not 2203?

Reply:
Or "International" and not either?

Reply:
2203 is a good in-joke for existing employees, but new interns aren't going to be as privy to our arcane naming structures.  It also leaves more wiggle room in case things change ;)  International definitely tempted me, but the original run was technically international since we had groups from all over north america participating (and a few interns in even more countries than that!)  So I figured I'd just keep the name time-based like it's previously been, and as unambiguous as I could convince myself what I have is!  I just have to make it a little different in the graphic so it doesn't run onto two lines.  Plus, the 2022 Edition sounds like something from a book title!